---
title: "&#39; Catch &#39; 블록에 도달할 수 없습니다. &lt;예외&gt; 윗부분에서 동일한 &#39; 처리 Try &#39; 문"
ms.date: 07/20/2015
ms.prod: .net
ms.technology:
- devlang-visual-basic
ms.topic: article
f1_keywords:
- bc42031
- vbc42031
helpviewer_keywords:
- BC42031
ms.assetid: 7d15597c-5988-42ea-a853-63cbf78faaf3
caps.latest.revision: 
author: dotnet-bot
ms.author: dotnetcontent
ms.openlocfilehash: b32e37f9dadaa43250906cf1d760f279754c60b0
ms.sourcegitcommit: c0dd436f6f8f44dc80dc43b07f6841a00b74b23f
ms.translationtype: MT
ms.contentlocale: ko-KR
ms.lasthandoff: 01/19/2018
---
# <a name="39catch39-block-never-reached-ltexceptiongt-handled-above-in-the-same-39try39-statement"></a>&#39; Catch &#39; 블록에 도달할 수 없습니다. &lt;예외&gt; 윗부분에서 동일한 &#39; 처리 Try &#39; 문
코드의 `Catch` 블록이 앞의 `Try` 블록에서 처리되므로 도달할 수 없습니다.  
  
 이 메시지는 기본적으로 경고입니다. 경고를 숨기 거 나 경고를 오류로 처리 하는 방법에 대 한 자세한 내용은 참조 하십시오 [Visual Basic에서 경고 구성](/visualstudio/ide/configuring-warnings-in-visual-basic)  
  
 **오류 ID:** BC42031  
  
## <a name="to-correct-this-error"></a>이 오류를 해결하려면  
  
-   중복 문을 제거합니다.  
  
## <a name="see-also"></a>참고 항목  
 [방법: Visual Basic에서 예외를 Catch 합니다.](http://msdn.microsoft.com/library/f3063c89-d2bf-49b1-91b5-b87edfb18b95)  
 [방법: 테스트 코드를 try... Visual Basic의 catch 블록](http://msdn.microsoft.com/library/8368e205-ed73-4185-a247-af84fb4fafa9)  
 [방법: Visual Basic의 Catch 블록에서 오류 필터링](http://msdn.microsoft.com/library/85964d0a-56e7-4301-a96e-5eaea23b7b9b)  
 [연습: 구조적 예외 처리 (Visual Basic)](http://msdn.microsoft.com/library/440da655-4b32-490b-8b16-bfe46f41fa76)  
 [Try...Catch...Finally 문](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)
